website/docs/guides/sl-snowflake-qs.md

```diff
@@ -0,0 +1,875 @@
+- **[Ratio metrics](/docs/build/metrics-overview#ratio-metrics):** Involve a numerator metric and a denominator metric. A constraint string can be applied to both the numerator and denominator or separately to the numerator or denominator.
+
+Once you've created your semantic models, it's time to start referencing those measures you made to create some metrics:
+
+1. Add metrics to your `fct_orders.yml` semantic model file:
+
+```yaml
+semantic_models:
+  - name: orders
+    defaults:
+      agg_time_dimension: order_date
+    description: |
+      Order fact table. This table’s grain is one row per order
+    model: ref('fct_orders')
+    entities:
+      - name: order_id
+        type: primary
+      - name: customer_id
+        type: foreign
+    dimensions:
+      - name: order_date
+        type: time
+        type_params:
+          time_granularity: day
+    measures:
+      - name: order_total
+        description: The total amount for each order including taxes.
+        agg: sum
+        expr: amount
+      - name: order_count
+        expr: 1
+        agg: sum
+      - name: customers_with_orders
+        description: Distinct count of customers placing orders
+        agg: count_distinct
+        expr: customer_id
+      - name: order_value_p99
+        expr: amount
+        agg: percentile
+        agg_params:
+          percentile: 0.99
+          use_discrete_percentile: True
+          use_approximate_percentile: False
+# Newly added          
+metrics: 
+  # Simple type metrics
+  - name: "order_total"
+    description: "Sum of orders value"
+    type: simple
+    label: "order_total"
+    type_params:
+      measure: order_total
+  - name: "order_count"
+    description: "number of orders"
+    type: simple
+    label: "order_count"
+    type_params:
+      measure: order_count
+  - name: large_orders
+    description: "Count of orders with order total over 20."
+    type: simple
+    label: "Large Orders"
+    type_params:
+      measure: order_count
+    filter: |
+      {{ Dimension('order_id__order_total_dim') }} >= 20
+  # Ratio type metric
+  - name: "avg_order_value"
+    label: "avg_order_value"
+    description: "average value of each order"
+    type: ratio
+    type_params:
+      numerator: order_total
+      denominator: order_count
+  # Cumulative type metrics
+  - name: "cumulative_order_amount_mtd"
+    label: "cumulative_order_amount_mtd"
+    description: "The month to date value of all orders"
+    type: cumulative
+    type_params:
+      measure: order_total
+      grain_to_date: month
+  # Derived metric
+  - name: "pct_of_orders_that_are_large"
+    label: "pct_of_orders_that_are_large"
+    description: "percent of orders that are large"
+    type: derived
+    type_params:
+      expr: large_orders/order_count
+      metrics:
+        - name: large_orders
+        - name: order_count
+```
+
```

Comment:
notes for me: might be good to add a screenshot to show what success looks like

![Screenshot 2024-04-12 at 17 15 52](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/10f1c084-7ab4-4ab7-9e3e-53db226ef9f0)